Update GCP and AWS docs on how to get Workload Identity (#2302)
astronomer/docs: astro/connect-gcp.md

To grant a Deployment on Astro access to external data services on GCP, such as BigQuery:

> 1. In the Cloud UI, select your Deployment, and then go to the **Details** tab of the Deployment.

Reviewer: ```suggestion
1. In the Cloud UI, select your Deployment, then click **Details**
```

> 2. Copy the service account shown under **Workload Identity**. This service account will be used in Step 3 to grant access.

Reviewer: ```suggestion
2. Copy the service account shown under **Workload Identity**.
```
